packages/runtime/src/agent/file_storage.ts

```diff
@@ -18,32 +29,33 @@ export class FileStorageAgent {
     return Deno.core.opAsync(
       "op_file_storage_agent_simple_put",
       {
-        url: url instanceof URL ? url.href : url,
+        path,
         ensureDir: options?.ensureDir ?? false,
+        configurationId: this.configurationId,
```

Comment:
I think it will be better to put configurationId before path and other arguments.